packages/io/svg-deserializer/shapesMapCsg.js

```diff
@@ -1,15 +1,13 @@
-const {CSG, CAG} = require('@jscad/csg')
-const {svg2cagX, svg2cagY, cagLengthX, cagLengthY, cagLengthP, reflect, groupValue} = require('./helpers')
-const {cssPxUnit} = require('./constants')
+const { geometry, primitives } = require('@jscad/csg')
```

Comment:
perhaps this file can be renamed to shapesMapGeomety ? or shapesMapInstanciate ?

Reply:
Done